Word.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+class Word 
+{
+    /** string $name **/
+    public $name;
+    /** string $translation **/
+    public $translation;
+    /** string $transcription **/
+    public $transcription;
+    /** string $description **/
+    public $description;
+    /** array $examples **/
+    public $examples;
+
+    public function save() {
```

Comment:
у методов тоже переносят скобку на новую строку

```suggestion
    public function save()
    {
```